static/js/apps/eval_rig/call_feedback.tsx

```diff
@@ -212,6 +226,7 @@ export function CallFeedback(): JSX.Element {
                   options={{
                     LLM_STAT_ACCURATE: "Stats seem accurate",
                     LLM_STAT_INACCURATE: "Stats seem inaccurate",
+                    LLM_STAT_NOTSURE: "Not sure",
```

Comment:
nit: Maybe make it symmetric to `DC_ANSWER_RELEVANT_UNSURE`?

LLM_STAT_UNSURE: "Unsure about accuracy"

Reply:
fixed